modules/Microsoft.Automation/automationAccounts/jobSchedules/deploy.bicep

```diff
@@ -1,5 +1,5 @@
 @description('Optional. Name of the Automation Account job schedule. Must be a GUID. If not provided, a new GUID is generated.')
-param name string = newGuid()
+param name string = uniqueString(automationAccountName, subscription().id)
```

Comment:
Unless something changed meanwhile, the proposed implementation would cause the deployment to fail if rerun. Ref https://docs.microsoft.com/en-us/azure/automation/shared-resources/schedules#create-a-schedule-with-a-resource-manager-template

```
Job schedule object name: A GUID (Globally Unique Identifier) is used as the name of the job schedule object.
Important
For each job schedule deployed with an ARM template, the GUID must be unique. Even if you're rescheduling an existing schedule, you'll need to change the GUID. This applies even if you've previously deleted an existing job schedule that was created with the same template. Reusing the same GUID results in a failed deployment.
```

Reply:
And here I was thinking this was the reason for the lack of idempotency with the component :D Will undo and disable the Bicep Linter warning. Thanks @eriqua

Reply:
Undid